lib/elixir/lib/calendar/date_range.ex

```diff
@@ -48,24 +50,43 @@ defmodule Date.Range do
       end
     end
 
-    def member?(_, _) do
+    def member?(%Date.Range{step: _}, _) do
       {:ok, false}
     end
 
+    # TODO: Remove me on v2.0
+    def member?(
+          %{__struct__: Date.Range, first_in_iso_days: first_days, last_in_iso_days: last_days} =
```

Comment:
```suggestion
          %Date.Range{first_in_iso_days: first_days, last_in_iso_days: last_days} =
```

Reply:
We are not pattern matching on range on purpose because those are "old versions" of the range. In theory the pattern works, because we don't check the size, but I would prefer to keep it explicit it is a subset.

Reply:
TIL, sorry for noise :)